### ISSUE TITLE
import error in LISTVIEW obj from VB6

### ISSUE THREAD

**Issue by Megaseso** [twinbasic/twinbasic#1769]
Hello. I had an error while importing Vb6.
I have already discovered the reason, but I think it is important to correct it.
These are the steps
Imported Listview control:
Private Sub LW_MouseMove(Button As Integer, Shift As Integer, x As Single, y As Single)
Fails. For it to work you must receive the value with byval
Private Sub LW_MouseMove(Button As Integer, Shift As Integer, ByVal x As Single, ByVal y As Single)
But if I insert the listview object from the application the parameters are different:
Private Sub LW_MouseMove(ByVal Button As Integer, ByVal Shift As Integer, ByVal x As OLE_XPOS_PIXELS, ByVal y As OLE_YPOS_PIXELS)
With these it works correctly without having to touch anything.

Greetings and very good work

**Comment by fafalone** [comment]
Both the 5.0 and 6.0 control are ByRef, so if that's not working it's a bug but not a problem with the definitions. Are you saying the 5.0/6.0 OCX controls work with ByVal?

The tB control, yes it should match that signature. It's still a work in progress, a lot of the features aren't implemented just quite yet. Thanks for pointing that out.

**Comment by Megaseso** [comment]
Yes, with byval they work correctly in both cases. What struck me as shocking is that when doing the conversion the way of passing the parameters was different, apart from the omission of the byval. I realized it when inserting the control from the menu. Apart from that bug, everything is perfect.

**Comment by fafalone** [comment]
It doesn't work in VB6; you get a signature mismatch error.

In tB it not only works but gives sane values... this is indeed very weird.
[scratch5582.zip](https://github.com/twinbasic/twinbasic/files/14224550/scratch5582.zip)

**Comment by fafalone** [comment]
This bug was fixed a while ago.
